### ISSUE TITLE
Clicking the "Test Config" button does nothing

### ISSUE THREAD

**Issue by rburgst** [MeetMe/jira-gerrit-plugin#15]
I have a problem with the plugin. It seems that it does not do anything. I configured everything but when I hit the test config button nothing happens. In Developer Tools I don't see any request being sent off. 
In the log file I don't see any entry (even though I configured Jira log levels for com.meetme to DEBUG).

**Comment by jhansche** [comment]
Hmm, not sure what's wrong with the Test Config button.  That is also not working for me currently, but the JIRA panel is still showing Gerrit reviews.  Can you confirm whether the JIRA panel is displaying reviews that match the given JIRA issue key?  If not, please confirm:

1. What is configured in "Issue Search" field?
2. What happens if you type that query, replacing `%s` with the JIRA issue key, into the top-right search box in Gerrit?  For example, if the JIRA key is "TEST-123" and the "Issue Search" field is "message:%s", then try typing "message:TEST-123" into the Gerrit search box.
3. Is JIRA configured to use a specialized gerrit account (i.e., not repurposing someone else's personal account, for example)?
4. On the Gerrit server, is there anything relevant in the `logs/sshd_log` or `logs/error_log` files?
5. From the JIRA server itself, are you able to ssh to your Gerrit server?  Can you ssh as the user you configured in JIRA?

**Comment by rburgst** [comment]
1. `message:%s`
2. I find the code review as expected
3. yes
4. nope, however there doesn't seem to be any recent entry in `logs/error_log` not sure what the problem here is, `sshd_log` doesn't show anything useful.
5. need to test this, haven't tried yet

In addition, I have turned on logging for `com.meetme` => `DEBUG` but I cannot find anything in the jira server logs.

**Comment by rburgst** [comment]
I made sure that I can reach and log into gerrit from the jira server. The test config button still does nothing.

**Comment by rburgst** [comment]
Ok, I managed to get it to work now, however, the TestConfig button really does nothing AFAICT.

My problem was that while I had configured the private key in gerrit and the gerrit jira plugin, I did not have the file in the jira users `.ssh` folder. Therefore trying to ssh to the gerrit server from the command line did not work.
By looking at the `sshd_log` and looking at a specific ticket in jira I could finally see that the jira server was actually trying to log in. 
After that I added the JIRA user to all the necessary groups and now it works.

It would be nice if the test config button actually tried a few of these things and gave some kind of feedback. This would help a lot when trying to find configuration issues.
